CHANGELOG.md

```diff
@@ -20,9 +20,9 @@ All notable changes to this project will be documented in this file.
 
 ### Dependency Upgrades
 
-- Upgrade to `eclipse-temurin:17.0.13_11-jre-focal`. (#162)
-- Upgrade to Spring Boot 3.0.13. (#162)
+- Upgrade to Java 17. (#162)
```

Comment:
You could have kept the message related to eclipse-temurin.
I did the change in libraries because there are no OCI images built upon eclipse-temurin in those cases.